### PR TITLE
Remove Redux From Shelf

### LINKED ISSUE #142: remove redux store code and dependencies (where applicable) from shelf component
## Type of Change
- [ ] New Feature Request
- [ ] Documentation / Website
- [ ] Improvement / Suggestion
- [ ] Bug
- [x] Other (please clarify below)

## Summary
In the shelf component for the website, there is [unused code connecting the component to a redux store](https://github.com/ProjectEvergreen/greenwood/blob/master/www/components/shelf/shelf.js#L18) that can be removed.

## Details
1. Remove the store connection code
1. Remove `redux` related dependencies (thunk?)
1. Remove other reference to `store.dispatch`
1. Should also be able to remove **pwa-helpers** as well (confirm is it can be removed from the `cli` package)

### PR DESCRIPTION
**Pull request by hutchgrant**
## Related Issue
Resolves #142 

## Summary of Changes

* Remove Redux From Shelf
* Fix tiny broken link in getting-started list

### CODE REVIEW

File: www/components/shelf/getting-started-list.json [hunk 1]
```diff
@@ -18,7 +18,7 @@
     }, {
       "index": 2,
       "name": "Project Structures",
-      "id": "project-structures"
+      "id": "project-structure"
```
Comment (thescientist13) [inline www/components/shelf/getting-started-list.json]: was it intentional to change the `id`?  Doesn't the `name` need to match to the `id` for in page scrolling?

Comment (hutchgrant) [reply www/components/shelf/getting-started-list.json]: the id has to match the header yes. That's why I changed it. It doesn't work currently.

Comment (thescientist13) [reply www/components/shelf/getting-started-list.json]: oh, the heading in the document.  got it.  So what is the `name` for?  If we're making everything else singular, shouldn't we change `name` too?

Comment (hutchgrant) [reply www/components/shelf/getting-started-list.json]: Name is for the text of the shelf link. We can also have be structure(singular) if you'd like but it has no affect on the link.

Comment (thescientist13) [reply www/components/shelf/getting-started-list.json]: I don't think it makes sense to have them be different then?  The left nav is one way, but the heading is another?  I would see that as a typo / grammar error.  

Can we make them all the same?

Comment (hutchgrant) [reply www/components/shelf/getting-started-list.json]: Singular or plural? Which is the more accurate heading name?  If plural, we can revert this change and fix the getting-started page for project-structure(s).  If singular, I need to add singular to the name of this shelf link.

Comment (thescientist13) [reply www/components/shelf/getting-started-list.json]: i think it was intended to be _Project Structure_ (singular)

**Pull request merged**